packages/block-library/src/page-list/convert-to-links-modal.js

```diff
@@ -4,8 +4,15 @@
 import { Button, Modal } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
-export const convertDescription = __(
-	'This page list is synced with the published pages on your site. Detach the page list to add, delete, or reorder pages yourself.'
+import { createInterpolateElement } from '@wordpress/element';
+
+export const convertDescription = createInterpolateElement(
+	__(
+		"The page list displays your site's published pages. If you choose to independently edit, your navigation will no longer <em>automatically</em> update when your pages change."
```

Comment:
What about something like this: 


The page list block displays your website's pages. Editing the block will allow you to add, delete, or reorder pages, although new pages will no longer be added automatically to this navigation.

Reply:
+1 to Rich's suggestion with a slight alternative to consider:

>The page list block displays all of your published pages as a menu, including any new pages. Editing this block allows you to add, delete, and reorder items but will prevent any new pages from being added going forward.

Reply:
Thanks for your input 👍 I appreciate the alternative ideas.

I like Rich's idea, but I think we should consider making the wording about what _will happen_ rather than describing the page list block.

Here's my slightly tweaked alternative which:

- describes the _current state_ of your menu.
- explains what proceeding with the proposed change will _enable_
- explains the _consequences_ of proceeding with the change

```
This menu is currently displaying your website's pages. Editing it will allow you to add, delete, or reorder pages, but new pages will no longer be automatically added to this navigation.
```

I think the key concept to get across to folks is that it will not _automatically_ update with changes to your pages. The word "automatic" is much more approachable than "sync" and everyone knows what it means.

> ...but will prevent any new pages from being added going forward.

Nit: Is this true? You _can_ easily add pages using the navigation link blocks. I'd prefer to avoid the word "prevent" as well.

Reply:
Updated to my suggestion [here](https://github.com/WordPress/gutenberg/pull/56829/commits/7b30fef50b60725f88f144b37ff4b9f318758113).

Still open for alternatives or we can merge and iterate... 🤷